.github/workflows/docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Build docs
+
+on:
+  push:
+    branches: [master]
+    tags: ["v*"]
+  release:
+    types: [created]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build_docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+      - name: install docs reqs
+        shell: bash -l {0}
+        run: |
+          pip install -U pip wheel
```

Comment:
I was also thinking to replace installation part with the one we do it with conda and cache... Your cache PR is still open, let me comment out there what have to be done...

Reply:
I would suggest using `actions/setup-python` as it's easier for caching. But it's up to you, conda or pip

Reply:
I agree with you and OK let's keep it simple here with `actions/setup-python`.